karmagrambot/analytics.py

```diff
@@ -12,3 +12,19 @@ def average_message_length(user_id, chat_id):
         return 0
 
     return sum(m['length'] for m in messages) / len(messages)
+
+def get_karma(user_id, chat_id):
+	db = dataset.connect(DB_URI)
```

Comment:
Wrong spacing (this line has a tab). This function is also undocumented and has no type hinting.